docs/application/dotnet/guides/opentk/window-of-opentk.md

```diff
@@ -0,0 +1,121 @@
+
+## Input Event
+
+In some app scenario, you want to do some actions while an app user input something. To do this, you can add event hander on the input event of `Window`. Also, you can use `key` and `mouse` event through using `Window` of `OpenTK` if a device supports both of them.
+
+```C#
+private IGameWindow mainWindow;
+
+protected override void OnCreate()
+{
+    mainWindow = Window;
+    
+    mainWindow.MouseMove += MouseMoveEventHandler;    // add event handler on mouse move event
+    mainWindow.KeyDown += KeyDownEventHandler;        // add event handler on key down event    
+}
+
+private void MouseMoveEventHandler(object sender, MouseMoveEventArgs e)
+{
+    if (e.Mouse[MouseButton.Left])      // get mouse move event which happening with click left button of mouse
+    {
+        float x = (float)(e.XDelta);    // the move distance in x-axis
+        float y = (float)(e.YDelta);    // the move distance in y-axis
+        // do your action
+    }
+    
+    if (e.Mouse[MouseButton.Right])     // get mouse move event which happening with click right button of mouse
+    {
+        float x = (float)(e.XDelta);    // the move distance in x-axis
+        float y = (float)(e.YDelta);    // the move distance in y-axis
+        // do your action
+    }
+}
+
+private void KeyDownEventHandler(object sender, KeyboardKeyEventArgs e)
+{
+    switch(e.Key)
+    {
+        case Key.Right:                 // press right key
+            // do your action
+            break;
+        case Key.Left:                  // press left key
+            // do your action
+            break;
+        case Key.Up:                    // press up key
+            // do your action
+            break;
+        case Key.Down:                  // press down key
+            // do your action
+            break;
+        case Key.Number0:               // press number 0 key
+            // do your action
+            break;
+        default:
+            break;
+    }
+}
+
+```
+
+## Window related Event
+
+`Window` also provides some window state related events such as `Closed`, `Closing`, `FocusedChanged`, `Resize`, `VisibleChanged`, `WindowStateChanged`, etc. In some scenario, you may want to mirror the state of `Window`, then 
+you can add event handlers on these events.
+
+```C#
+private IGameWindow mainWindow;
+
+protected override void OnCreate()
+{
+    mainWindow = Window;
+    
+    mainWindow.Resize += ResizeEventHandler;    // add event handler on window resize event
+}
+
+private void ResizeEventHandler(object sender, EventArgs e)
+{
+    // do your action
+}
+```
+
+## Render related Event
+
+`Window` also provides the render ralated events, including: `UpdateFrame`, `RenderFrame`.
+`RenderFrame` event occurs when it is time to render a frame; `UpdateFrame` event occurs when a frame is updated
+before `RenderFrame` event occurs. These events will occur repeatedly while OpenTK's main loop is running.
+```C#
```

Comment:
Please change to:
`Window` also provides the render related events including the following: 

- `UpdateFrame` : This event occurs when a frame is updated.
- `RenderFrame `: This event occurs when a  frame is updated before `RenderFrame` event occurs. 

These events will occur repeatedly while OpenTK's main loop is running.